create_infotexts.py

```diff
@@ -371,7 +371,11 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
         place_as_wikitext = ""
 
         if place_string:
-            if place_string in places_mapping:
+            # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
+            if places_mapping[place_string].get('commonscat'):
```

Comment:
This re-jig doesn't work.

In the previous version we first checked that place_string  was in places_mapping. If this was the case we then also checked for a commonscat. Here `places_mapping[place_string]` is likely to throw a `keyError`.

To fix re-add `if place_string in places_mapping:` before this (see also next comment).